### ISSUE TITLE
InfluxDB error "packetLoss"

### ISSUE THREAD

**Issue by guigui42** [alexjustesen/speedtest-tracker#301]
InfluxDB version : 2.6.1
Speedtest-tracker version : v0.9.2

all on docker, using traefik, I already use InfluxDB (using port 443) for Home Assistant, so this already works fine.

![image](https://user-images.githubusercontent.com/2376010/211155906-a31b1778-394c-443e-a2d4-d15e7e860561.png)

I tried adding :443 in the URL, I already tried using the OrgID instead of the OrgName, but same result : 

```
01/07/2023 3:27:05 PM
[2023-01-07 14:27:05] production.ERROR: Undefined array key "packetLoss" {"exception":"[object] (ErrorException(code: 0): Undefined array key \"packetLoss\" at /var/www/html/app/Models/Result.php:78)
01/07/2023 3:27:05 PM
[stacktrace]
01/07/2023 3:27:05 PM
#0 /var/www/html/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(266): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleError()
01/07/2023 3:27:05 PM
#1 /var/www/html/app/Models/Result.php(78): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->Illuminate\\Foundation\\Bootstrap\\{closure}()
01/07/2023 3:27:05 PM
#2 /var/www/html/app/Jobs/SendDataToInfluxDbV2.php(63): App\\Models\\Result->formatForInfluxDB2()
01/07/2023 3:27:05 PM
#3 /var/www/html/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(36): App\\Jobs\\SendDataToInfluxDbV2->handle()
01/07/2023 3:27:05 PM
#4 /var/www/html/vendor/laravel/framework/src/Illuminate/Container/Util.php(41): Illuminate\\Container\\BoundMethod::Illuminate\\Container\\{closure}()
01/07/2023 3:27:05 PM
#5 /var/www/html/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(93): Illuminate\\Container\\Util::unwrapIfClosure()
01/07/2023 3:27:05 PM
#6 /var/www/html/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(37): Illuminate\\Container\\BoundMethod::callBoundMethod()
01/07/2023 3:27:05 PM
#7 /var/www/html/vendor/laravel/framework/src/Illuminate/Container/Container.php(651): Illuminate\\Container\\BoundMethod::call()
01/07/2023 3:27:05 PM
#8 /var/www/html/vendor/laravel/framework/src/Illuminate/Bus/Dispatcher.php(128): Illuminate\\Container\\Container->call()
01/07/2023 3:27:05 PM
#9 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(141): Illuminate\\Bus\\Dispatcher->Illuminate\\Bus\\{closure}()
01/07/2023 3:27:05 PM
#10 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(116): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
01/07/2023 3:27:05 PM
#11 /var/www/html/vendor/laravel/framework/src/Illuminate/Bus/Dispatcher.php(132): Illuminate\\Pipeline\\Pipeline->then()
01/07/2023 3:27:05 PM
#12 /var/www/html/vendor/laravel/framework/src/Illuminate/Queue/CallQueuedHandler.php(124): Illuminate\\Bus\\Dispatcher->dispatchNow()
01/07/2023 3:27:05 PM
#13 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(141): Illuminate\\Queue\\CallQueuedHandler->Illuminate\\Queue\\{closure}()
01/07/2023 3:27:05 PM
#14 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(116): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
01/07/2023 3:27:05 PM
#15 /var/www/html/vendor/laravel/framework/src/Illuminate/Queue/CallQueuedHandler.php(126): Illuminate\\Pipeline\\Pipeline->then()
01/07/2023 3:27:05 PM
#16 /var/www/html/vendor/laravel/framework/src/Illuminate/Queue/CallQueuedHandler.php(70): Illuminate\\Queue\\CallQueuedHandler->dispatchThroughMiddleware()
01/07/2023 3:27:05 PM
#17 /var/www/html/vendor/laravel/framework/src/Illuminate/Queue/Jobs/Job.php(98): Illuminate\\Queue\\CallQueuedHandler->call()
01/07/2023 3:27:05 PM
#18 /var/www/html/vendor/laravel/framework/src/Illuminate/Queue/Worker.php(425): Illuminate\\Queue\\Jobs\\Job->fire()
01/07/2023 3:27:05 PM
#19 /var/www/html/vendor/laravel/framework/src/Illuminate/Queue/Worker.php(375): Illuminate\\Queue\\Worker->process()
01/07/2023 3:27:05 PM
#20 /var/www/html/vendor/laravel/framework/src/Illuminate/Queue/Worker.php(173): Illuminate\\Queue\\Worker->runJob()
01/07/2023 3:27:05 PM
#21 /var/www/html/vendor/laravel/framework/src/Illuminate/Queue/Console/WorkCommand.php(147): Illuminate\\Queue\\Worker->daemon()
01/07/2023 3:27:05 PM
#22 /var/www/html/vendor/laravel/framework/src/Illuminate/Queue/Console/WorkCommand.php(130): Illuminate\\Queue\\Console\\WorkCommand->runWorker()
01/07/2023 3:27:05 PM
#23 /var/www/html/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(36): Illuminate\\Queue\\Console\\WorkCommand->handle()
01/07/2023 3:27:05 PM
#24 /var/www/html/vendor/laravel/framework/src/Illuminate/Container/Util.php(41): Illuminate\\Container\\BoundMethod::Illuminate\\Container\\{closure}()
01/07/2023 3:27:05 PM
#25 /var/www/html/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(93): Illuminate\\Container\\Util::unwrapIfClosure()
01/07/2023 3:27:05 PM
#26 /var/www/html/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(37): Illuminate\\Container\\BoundMethod::callBoundMethod()
01/07/2023 3:27:05 PM
#27 /var/www/html/vendor/laravel/framework/src/Illuminate/Container/Container.php(651): Illuminate\\Container\\BoundMethod::call()
01/07/2023 3:27:05 PM
#28 /var/www/html/vendor/laravel/framework/src/Illuminate/Console/Command.php(182): Illuminate\\Container\\Container->call()
01/07/2023 3:27:05 PM
#29 /var/www/html/vendor/symfony/console/Command/Command.php(312): Illuminate\\Console\\Command->execute()
01/07/2023 3:27:05 PM
#30 /var/www/html/vendor/laravel/framework/src/Illuminate/Console/Command.php(152): Symfony\\Component\\Console\\Command\\Command->run()
01/07/2023 3:27:05 PM
#31 /var/www/html/vendor/symfony/console/Application.php(1022): Illuminate\\Console\\Command->run()
01/07/2023 3:27:05 PM
#32 /var/www/html/vendor/symfony/console/Application.php(314): Symfony\\Component\\Console\\Application->doRunCommand()
01/07/2023 3:27:05 PM
#33 /var/www/html/vendor/symfony/console/Application.php(168): Symfony\\Component\\Console\\Application->doRun()
01/07/2023 3:27:05 PM
#34 /var/www/html/vendor/laravel/framework/src/Illuminate/Console/Application.php(102): Symfony\\Component\\Console\\Application->run()
01/07/2023 3:27:05 PM
#35 /var/www/html/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(155): Illuminate\\Console\\Application->run()
01/07/2023 3:27:05 PM
#36 /var/www/html/artisan(37): Illuminate\\Foundation\\Console\\Kernel->handle()
01/07/2023 3:27:05 PM
#37 {main}
01/07/2023 3:27:05 PM
"} 
01/07/2023 3:27:05 PM
[2023-01-07 14:27:05] production.ERROR: Undefined array key "packetLoss" {"exception":"[object] (ErrorException(code: 0): Undefined array key \"packetLoss\" at /var/www/html/app/Models/Result.php:78)
01/07/2023 3:27:05 PM
[stacktrace]
01/07/2023 3:27:05 PM
#0 /var/www/html/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(266): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleError()
01/07/2023 3:27:05 PM
#1 /var/www/html/app/Models/Result.php(78): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->Illuminate\\Foundation\\Bootstrap\\{closure}()
01/07/2023 3:27:05 PM
#2 /var/www/html/app/Jobs/SendDataToInfluxDbV2.php(63): App\\Models\\Result->formatForInfluxDB2()
01/07/2023 3:27:05 PM
#3 /var/www/html/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(36): App\\Jobs\\SendDataToInfluxDbV2->handle()
01/07/2023 3:27:05 PM
#4 /var/www/html/vendor/laravel/framework/src/Illuminate/Container/Util.php(41): Illuminate\\Container\\BoundMethod::Illuminate\\Container\\{closure}()
01/07/2023 3:27:05 PM
#5 /var/www/html/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(93): Illuminate\\Container\\Util::unwrapIfClosure()
01/07/2023 3:27:05 PM
#6 /var/www/html/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(37): Illuminate\\Container\\BoundMethod::callBoundMethod()
01/07/2023 3:27:05 PM
#7 /var/www/html/vendor/laravel/framework/src/Illuminate/Container/Container.php(651): Illuminate\\Container\\BoundMethod::call()
01/07/2023 3:27:05 PM
#8 /var/www/html/vendor/laravel/framework/src/Illuminate/Bus/Dispatcher.php(128): Illuminate\\Container\\Container->call()
01/07/2023 3:27:05 PM
#9 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(141): Illuminate\\Bus\\Dispatcher->Illuminate\\Bus\\{closure}()
01/07/2023 3:27:05 PM
#10 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(116): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
01/07/2023 3:27:05 PM
#11 /var/www/html/vendor/laravel/framework/src/Illuminate/Bus/Dispatcher.php(132): Illuminate\\Pipeline\\Pipeline->then()
01/07/2023 3:27:05 PM
#12 /var/www/html/vendor/laravel/framework/src/Illuminate/Queue/CallQueuedHandler.php(124): Illuminate\\Bus\\Dispatcher->dispatchNow()
01/07/2023 3:27:05 PM
#13 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(141): Illuminate\\Queue\\CallQueuedHandler->Illuminate\\Queue\\{closure}()
01/07/2023 3:27:05 PM
#14 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(116): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
01/07/2023 3:27:05 PM
#15 /var/www/html/vendor/laravel/framework/src/Illuminate/Queue/CallQueuedHandler.php(126): Illuminate\\Pipeline\\Pipeline->then()
01/07/2023 3:27:05 PM
#16 /var/www/html/vendor/laravel/framework/src/Illuminate/Queue/CallQueuedHandler.php(70): Illuminate\\Queue\\CallQueuedHandler->dispatchThroughMiddleware()
01/07/2023 3:27:05 PM
#17 /var/www/html/vendor/laravel/framework/src/Illuminate/Queue/Jobs/Job.php(98): Illuminate\\Queue\\CallQueuedHandler->call()
01/07/2023 3:27:05 PM
#18 /var/www/html/vendor/laravel/framework/src/Illuminate/Queue/Worker.php(425): Illuminate\\Queue\\Jobs\\Job->fire()
01/07/2023 3:27:05 PM
#19 /var/www/html/vendor/laravel/framework/src/Illuminate/Queue/Worker.php(375): Illuminate\\Queue\\Worker->process()
01/07/2023 3:27:05 PM
#20 /var/www/html/vendor/laravel/framework/src/Illuminate/Queue/Worker.php(173): Illuminate\\Queue\\Worker->runJob()
01/07/2023 3:27:05 PM
#21 /var/www/html/vendor/laravel/framework/src/Illuminate/Queue/Console/WorkCommand.php(147): Illuminate\\Queue\\Worker->daemon()
01/07/2023 3:27:05 PM
#22 /var/www/html/vendor/laravel/framework/src/Illuminate/Queue/Console/WorkCommand.php(130): Illuminate\\Queue\\Console\\WorkCommand->runWorker()
01/07/2023 3:27:05 PM
#23 /var/www/html/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(36): Illuminate\\Queue\\Console\\WorkCommand->handle()
01/07/2023 3:27:05 PM
#24 /var/www/html/vendor/laravel/framework/src/Illuminate/Container/Util.php(41): Illuminate\\Container\\BoundMethod::Illuminate\\Container\\{closure}()
01/07/2023 3:27:05 PM
#25 /var/www/html/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(93): Illuminate\\Container\\Util::unwrapIfClosure()
01/07/2023 3:27:05 PM
#26 /var/www/html/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(37): Illuminate\\Container\\BoundMethod::callBoundMethod()
01/07/2023 3:27:05 PM
#27 /var/www/html/vendor/laravel/framework/src/Illuminate/Container/Container.php(651): Illuminate\\Container\\BoundMethod::call()
01/07/2023 3:27:05 PM
#28 /var/www/html/vendor/laravel/framework/src/Illuminate/Console/Command.php(182): Illuminate\\Container\\Container->call()
01/07/2023 3:27:05 PM
#29 /var/www/html/vendor/symfony/console/Command/Command.php(312): Illuminate\\Console\\Command->execute()
01/07/2023 3:27:05 PM
#30 /var/www/html/vendor/laravel/framework/src/Illuminate/Console/Command.php(152): Symfony\\Component\\Console\\Command\\Command->run()
01/07/2023 3:27:05 PM
#31 /var/www/html/vendor/symfony/console/Application.php(1022): Illuminate\\Console\\Command->run()
01/07/2023 3:27:05 PM
#32 /var/www/html/vendor/symfony/console/Application.php(314): Symfony\\Component\\Console\\Application->doRunCommand()
01/07/2023 3:27:05 PM
#33 /var/www/html/vendor/symfony/console/Application.php(168): Symfony\\Component\\Console\\Application->doRun()
01/07/2023 3:27:05 PM
#34 /var/www/html/vendor/laravel/framework/src/Illuminate/Console/Application.php(102): Symfony\\Component\\Console\\Application->run()
01/07/2023 3:27:05 PM
#35 /var/www/html/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(155): Illuminate\\Console\\Application->run()
01/07/2023 3:27:05 PM
#36 /var/www/html/artisan(37): Illuminate\\Foundation\\Console\\Kernel->handle()
01/07/2023 3:27:05 PM
#37 {main}
01/07/2023 3:27:05 PM
"} 
01/07/2023 3:27:05 PM
[2023-01-07 14:27:05] production.ERROR: Undefined array key "packetLoss" {"exception":"[object] (ErrorException(code: 0): Undefined array key \"packetLoss\" at /var/www/html/app/Models/Result.php:78)
01/07/2023 3:27:05 PM
[stacktrace]
01/07/2023 3:27:05 PM
#0 /var/www/html/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(266): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleError()
01/07/2023 3:27:05 PM
#1 /var/www/html/app/Models/Result.php(78): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->Illuminate\\Foundation\\Bootstrap\\{closure}()
01/07/2023 3:27:05 PM
#2 /var/www/html/app/Jobs/SendDataToInfluxDbV2.php(63): App\\Models\\Result->formatForInfluxDB2()
01/07/2023 3:27:05 PM
#3 /var/www/html/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(36): App\\Jobs\\SendDataToInfluxDbV2->handle()
01/07/2023 3:27:05 PM
#4 /var/www/html/vendor/laravel/framework/src/Illuminate/Container/Util.php(41): Illuminate\\Container\\BoundMethod::Illuminate\\Container\\{closure}()
01/07/2023 3:27:05 PM
#5 /var/www/html/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(93): Illuminate\\Container\\Util::unwrapIfClosure()
01/07/2023 3:27:05 PM
#6 /var/www/html/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(37): Illuminate\\Container\\BoundMethod::callBoundMethod()
01/07/2023 3:27:05 PM
#7 /var/www/html/vendor/laravel/framework/src/Illuminate/Container/Container.php(651): Illuminate\\Container\\BoundMethod::call()
01/07/2023 3:27:05 PM
#8 /var/www/html/vendor/laravel/framework/src/Illuminate/Bus/Dispatcher.php(128): Illuminate\\Container\\Container->call()
01/07/2023 3:27:05 PM
#9 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(141): Illuminate\\Bus\\Dispatcher->Illuminate\\Bus\\{closure}()
01/07/2023 3:27:05 PM
#10 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(116): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
01/07/2023 3:27:05 PM
#11 /var/www/html/vendor/laravel/framework/src/Illuminate/Bus/Dispatcher.php(132): Illuminate\\Pipeline\\Pipeline->then()
01/07/2023 3:27:05 PM
#12 /var/www/html/vendor/laravel/framework/src/Illuminate/Queue/CallQueuedHandler.php(124): Illuminate\\Bus\\Dispatcher->dispatchNow()
01/07/2023 3:27:05 PM
#13 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(141): Illuminate\\Queue\\CallQueuedHandler->Illuminate\\Queue\\{closure}()
01/07/2023 3:27:05 PM
#14 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(116): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
01/07/2023 3:27:05 PM
#15 /var/www/html/vendor/laravel/framework/src/Illuminate/Queue/CallQueuedHandler.php(126): Illuminate\\Pipeline\\Pipeline->then()
01/07/2023 3:27:05 PM
#16 /var/www/html/vendor/laravel/framework/src/Illuminate/Queue/CallQueuedHandler.php(70): Illuminate\\Queue\\CallQueuedHandler->dispatchThroughMiddleware()
01/07/2023 3:27:05 PM
#17 /var/www/html/vendor/laravel/framework/src/Illuminate/Queue/Jobs/Job.php(98): Illuminate\\Queue\\CallQueuedHandler->call()
01/07/2023 3:27:05 PM
#18 /var/www/html/vendor/laravel/framework/src/Illuminate/Queue/Worker.php(425): Illuminate\\Queue\\Jobs\\Job->fire()
01/07/2023 3:27:05 PM
#19 /var/www/html/vendor/laravel/framework/src/Illuminate/Queue/Worker.php(375): Illuminate\\Queue\\Worker->process()
01/07/2023 3:27:05 PM
#20 /var/www/html/vendor/laravel/framework/src/Illuminate/Queue/Worker.php(173): Illuminate\\Queue\\Worker->runJob()
01/07/2023 3:27:05 PM
#21 /var/www/html/vendor/laravel/framework/src/Illuminate/Queue/Console/WorkCommand.php(147): Illuminate\\Queue\\Worker->daemon()
01/07/2023 3:27:05 PM
#22 /var/www/html/vendor/laravel/framework/src/Illuminate/Queue/Console/WorkCommand.php(130): Illuminate\\Queue\\Console\\WorkCommand->runWorker()
01/07/2023 3:27:05 PM
#23 /var/www/html/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(36): Illuminate\\Queue\\Console\\WorkCommand->handle()
01/07/2023 3:27:05 PM
#24 /var/www/html/vendor/laravel/framework/src/Illuminate/Container/Util.php(41): Illuminate\\Container\\BoundMethod::Illuminate\\Container\\{closure}()
01/07/2023 3:27:05 PM
#25 /var/www/html/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(93): Illuminate\\Container\\Util::unwrapIfClosure()
01/07/2023 3:27:05 PM
#26 /var/www/html/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(37): Illuminate\\Container\\BoundMethod::callBoundMethod()
01/07/2023 3:27:05 PM
#27 /var/www/html/vendor/laravel/framework/src/Illuminate/Container/Container.php(651): Illuminate\\Container\\BoundMethod::call()
01/07/2023 3:27:05 PM
#28 /var/www/html/vendor/laravel/framework/src/Illuminate/Console/Command.php(182): Illuminate\\Container\\Container->call()
01/07/2023 3:27:05 PM
#29 /var/www/html/vendor/symfony/console/Command/Command.php(312): Illuminate\\Console\\Command->execute()
01/07/2023 3:27:05 PM
#30 /var/www/html/vendor/laravel/framework/src/Illuminate/Console/Command.php(152): Symfony\\Component\\Console\\Command\\Command->run()
01/07/2023 3:27:05 PM
#31 /var/www/html/vendor/symfony/console/Application.php(1022): Illuminate\\Console\\Command->run()
01/07/2023 3:27:05 PM
#32 /var/www/html/vendor/symfony/console/Application.php(314): Symfony\\Component\\Console\\Application->doRunCommand()
01/07/2023 3:27:05 PM
#33 /var/www/html/vendor/symfony/console/Application.php(168): Symfony\\Component\\Console\\Application->doRun()
01/07/2023 3:27:05 PM
#34 /var/www/html/vendor/laravel/framework/src/Illuminate/Console/Application.php(102): Symfony\\Component\\Console\\Application->run()
01/07/2023 3:27:05 PM
#35 /var/www/html/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(155): Illuminate\\Console\\Application->run()
01/07/2023 3:27:05 PM
#36 /var/www/html/artisan(37): Illuminate\\Foundation\\Console\\Kernel->handle()
01/07/2023 3:27:05 PM
#37 {main}
01/07/2023 3:27:05 PM
"} 
```

**Comment by alexdelprete** [comment]
Are you sure the bucket is a new one and not an old one made for the old ST project?

**Comment by guigui42** [comment]
Ok I figured out the solution : I had to use the OrgID and not the org name (I thought I had tried before, but didnt work).

Anyway, now this works : 
![image](https://user-images.githubusercontent.com/2376010/211168238-31eb3d5b-8545-4908-a902-47fcb1df643b.png)

Maybe there could be some better error handling, or updated doc ?

Here is my Org profile in InfluxDB for information : 
![image](https://user-images.githubusercontent.com/2376010/211168345-e465d204-483f-48cb-a817-ffd20ca04bd2.png)

**Comment by guigui42** [comment]
Nevermind, might not be so easy, it seems to work "sometimes" only ...

```
01/07/2023 9:13:19 PM
[2023-01-07 20:13:19] production.ERROR: Undefined array key "packetLoss" {"exception":"[object] (ErrorException(code: 0): Undefined array key \"packetLoss\" at /var/www/html/app/Models/Result.php:78)
```

After some more digging, the error appears only when using some specific Servers :  
![image](https://user-images.githubusercontent.com/2376010/211168973-0f0761cf-c3e4-4211-a0b8-85dcb08ac1f9.png)

and in InfluxDB, the only ones working are the ones from the "KEYYO" server : 
![image](https://user-images.githubusercontent.com/2376010/211168994-1af0c40a-c220-4fad-8878-c09ad1233024.png)

But other servers like fdcservers.net are not working when sending the data to InfluxDB for some reason, maybe the value of the "packetLoss" data ?

**Comment by alexdelprete** [comment]
It's not the OrgID, you should put `home` in that field.

> But other servers like fdcservers.net are not working when sending the data to InfluxDB for some reason, maybe the value of the "packetLoss" data ?

unfortunately not all servers provide all the same data, probably ST is not validating some data when exporting to influxdb. Let's wait for @alexjustesen to verify this.

**Comment by StephenShamakian** [comment]
I am having the same problem with a brand new install of speedtest-tracker. I also have a custom Org name in InfluxDB. I tried both the Org Name and the Org ID. I am getting this regardless:

`[2023-01-09 22:48:56] production.ERROR: Undefined array key "packetLoss" {"exception":"[object] (ErrorException(code: 0): Undefined array key \"packetLoss\" at /var/www/html/app/Models/Result.php:78)
[stacktrace]
#0 /var/www/html/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(266): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleError()
#1 /var/www/html/app/Models/Result.php(78): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->Illuminate\\Foundation\\Bootstrap\\{closure}()
#2 /var/www/html/app/Jobs/SendDataToInfluxDbV2.php(63): App\\Models\\Result->formatForInfluxDB2()
#3 /var/www/html/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(36): App\\Jobs\\SendDataToInfluxDbV2->handle()
#4 /var/www/html/vendor/laravel/framework/src/Illuminate/Container/Util.php(41): Illuminate\\Container\\BoundMethod::Illuminate\\Container\\{closure}()
#5 /var/www/html/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(93): Illuminate\\Container\\Util::unwrapIfClosure()
#6 /var/www/html/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(37): Illuminate\\Container\\BoundMethod::callBoundMethod()
#7 /var/www/html/vendor/laravel/framework/src/Illuminate/Container/Container.php(651): Illuminate\\Container\\BoundMethod::call()
#8 /var/www/html/vendor/laravel/framework/src/Illuminate/Bus/Dispatcher.php(128): Illuminate\\Container\\Container->call()
#9 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(141): Illuminate\\Bus\\Dispatcher->Illuminate\\Bus\\{closure}()
#10 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(116): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#11 /var/www/html/vendor/laravel/framework/src/Illuminate/Bus/Dispatcher.php(132): Illuminate\\Pipeline\\Pipeline->then()
#12 /var/www/html/vendor/laravel/framework/src/Illuminate/Queue/CallQueuedHandler.php(124): Illuminate\\Bus\\Dispatcher->dispatchNow()
#13 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(141): Illuminate\\Queue\\CallQueuedHandler->Illuminate\\Queue\\{closure}()
#14 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(116): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#15 /var/www/html/vendor/laravel/framework/src/Illuminate/Queue/CallQueuedHandler.php(126): Illuminate\\Pipeline\\Pipeline->then()
#16 /var/www/html/vendor/laravel/framework/src/Illuminate/Queue/CallQueuedHandler.php(70): Illuminate\\Queue\\CallQueuedHandler->dispatchThroughMiddleware()
#17 /var/www/html/vendor/laravel/framework/src/Illuminate/Queue/Jobs/Job.php(98): Illuminate\\Queue\\CallQueuedHandler->call()
#18 /var/www/html/vendor/laravel/framework/src/Illuminate/Queue/Worker.php(425): Illuminate\\Queue\\Jobs\\Job->fire()
#19 /var/www/html/vendor/laravel/framework/src/Illuminate/Queue/Worker.php(375): Illuminate\\Queue\\Worker->process()
#20 /var/www/html/vendor/laravel/framework/src/Illuminate/Queue/Worker.php(173): Illuminate\\Queue\\Worker->runJob()
#21 /var/www/html/vendor/laravel/framework/src/Illuminate/Queue/Console/WorkCommand.php(147): Illuminate\\Queue\\Worker->daemon()
#22 /var/www/html/vendor/laravel/framework/src/Illuminate/Queue/Console/WorkCommand.php(130): Illuminate\\Queue\\Console\\WorkCommand->runWorker()
#23 /var/www/html/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(36): Illuminate\\Queue\\Console\\WorkCommand->handle()
#24 /var/www/html/vendor/laravel/framework/src/Illuminate/Container/Util.php(41): Illuminate\\Container\\BoundMethod::Illuminate\\Container\\{closure}()
#25 /var/www/html/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(93): Illuminate\\Container\\Util::unwrapIfClosure()
#26 /var/www/html/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(37): Illuminate\\Container\\BoundMethod::callBoundMethod()
#27 /var/www/html/vendor/laravel/framework/src/Illuminate/Container/Container.php(651): Illuminate\\Container\\BoundMethod::call()
#28 /var/www/html/vendor/laravel/framework/src/Illuminate/Console/Command.php(182): Illuminate\\Container\\Container->call()
#29 /var/www/html/vendor/symfony/console/Command/Command.php(312): Illuminate\\Console\\Command->execute()
#30 /var/www/html/vendor/laravel/framework/src/Illuminate/Console/Command.php(152): Symfony\\Component\\Console\\Command\\Command->run()
#31 /var/www/html/vendor/symfony/console/Application.php(1022): Illuminate\\Console\\Command->run()
#32 /var/www/html/vendor/symfony/console/Application.php(314): Symfony\\Component\\Console\\Application->doRunCommand()
#33 /var/www/html/vendor/symfony/console/Application.php(168): Symfony\\Component\\Console\\Application->doRun()
#34 /var/www/html/vendor/laravel/framework/src/Illuminate/Console/Application.php(102): Symfony\\Component\\Console\\Application->run()
#35 /var/www/html/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(155): Illuminate\\Console\\Application->run()
#36 /var/www/html/artisan(37): Illuminate\\Foundation\\Console\\Kernel->handle()
#37 {main}
"} `

**Comment by alexjustesen** [comment]
Yeah this is because not all servers return the same data it seems. It's likely companies who are running older version of the speedtest software that doesn't capture the packet loss.

I've got this on my list 👍

**Comment by StephenShamakian** [comment]
@alexjustesen if you want an example server, I'm using the Comcast Massachusetts server (ID: 1774)

**Comment by alexjustesen** [comment]
> @alexjustesen if you want an example server, I'm using the Comcast Massachusetts server (ID: 1774)

Guess I probably should seeing how we're in the same state. Thanks for the test server suggestion. I'll see if I can break it.

**Comment by alexjustesen** [comment]
> Small world! I have co-workers that used to work for MassMutual! I also used to be on the Web Engineering team at Aetna as well.

Small world indeed, might have to keep in touch

**Comment by alexjustesen** [comment]
Moving this to milestone `v0.13.0` as this method will be refactored during the data fix.

**Comment by PovilasID** [comment]
Hey I am also seeing only data from some servers show up. Here is an a server that does not get passed: 16248

**Comment by alexjustesen** [comment]
closed in #486
